examples/stm32/nucleo-h723zg-make-baremetal-builtin/link.ld

```diff
@@ -0,0 +1,34 @@
+ENTRY(Reset_Handler);
+MEMORY {
+  flash(rx) : ORIGIN = 0x08000000, LENGTH = 1024k
+  sram(rwx) : ORIGIN = 0x24000000, LENGTH = 256k  /* AXI SRAM in domain D1 */
+  /* 2.3.2: remaining SRAM is in other (non-contiguous) banks,
+     DTCM @0x20000000 is in domain D1 and not accessible by the ETH DMA controller in domain D2
+     SRAM @0x30000000 is in domain D2 and not directly available at startup to be used as stack (8.5.9 page 366) */
+}
+_estack     = ORIGIN(sram) + LENGTH(sram);    /* stack points to end of SRAM */
+
+SECTIONS {
+  .vectors  : { KEEP(*(.isr_vector)) }  > flash
+  .text     : { *(.text* .text.*) }     > flash
+  .rodata   : { *(.rodata*) }           > flash
+
+  .data : {
+    _sdata = .;   /* for init_ram() */
+    *(.first_data)
+    *(.data SORT(.data.*))
+    *(.bootloader)        /* .bootloader function sections */
```

Comment:
The common practice is to call this section "iram" (Instruction RAM) - let's use `.iram` instead

Reply:
Yes, "bootloader" I though of a specific flash partition